modules.d/90mdraid/module-setup.sh

```diff
@@ -89,7 +89,7 @@ install() {
         rule_path="${initdir}${udevdir}/rules.d/${rule}"
         # shellcheck disable=SC2016
         [ -f "${rule_path}" ] && sed -i -r \
-            -e '/(RUN|IMPORT\{program\})\+?="[[:alpha:]/]*mdadm[[:blank:]]+(--incremental|-I)[[:blank:]]+(--export )?(\$env\{DEVNAME\}|\$tempnode|\$devnode)/d' \
+            -e '/(RUN|IMPORT\{program\})\+?="[[:alpha:]]*mdadm[[:blank:]]+(--incremental|-I)[[:blank:]]+(--export )?(\$env\{DEVNAME\}|\$tempnode|\$devnode)/d' \
```

Comment:
This change looks incorrect, to me.

`[[:alpha:]/]` creates a character list with the contents of `[:alpha:]` as well as the character `/` in order to match path prefixes for the `mdadm` command. Removing the slash from the list will make it match things like `foomdadm` instead of the intended `/foo/mdadm`.

For example...
```
~> echo 'blah blah
         blah blah
         RUN="/foo/mdadm bar baz"
         blah blah' | gsed -re '/(RUN|IMPORT)="[[:alpha:]]*mdadm[[:blank:]/]+/ s/mdadm/MDADM/'
blah blah
blah blah
RUN="/foo/mdadm bar baz"
blah blah
~> echo 'blah blah
         blah blah
         RUN="/foo/mdadm bar baz"
         blah blah' | gsed -re '/(RUN|IMPORT)="[[:alpha:]/]*mdadm[[:blank:]]+/ s/mdadm/MDADM/'
blah blah
blah blah
RUN="/foo/MDADM bar baz"
blah blah
```

Reply:
Right, the new code does not remove the lines with mdadm with the incremental options. E.g.:
```
# grep -e "\-\-incremental " -e "\-I " /usr/lib/udev/rules.d/64-md-raid-assembly.rules
ACTION=="add|change", IMPORT{program}="/usr/sbin/mdadm --incremental --export $devnode --offroot $env{DEVLINKS}"
# sed -n -r -e '/(RUN|IMPORT\{program\})\+?="[[:alpha:]/]*mdadm[[:blank:]]+(--incremental|-I)[[:blank:]]+(--export )?(\$env\{DEVNAME\}|\$tempnode|\$devnode)/p' /usr/lib/udev/rules.d/64-md-raid-assembly.rules
ACTION=="add|change", IMPORT{program}="/usr/sbin/mdadm --incremental --export $devnode --offroot $env{DEVLINKS}"
# sed -n -r -e '/(RUN|IMPORT\{program\})\+?="[[:alpha:]]*mdadm[[:blank:]]+(--incremental|-I)[[:blank:]]+(--export )?(\$env\{DEVNAME\}|\$tempnode|\$devnode)/p' /usr/lib/udev/rules.d/64-md-raid-assembly.rules
```